spec-tests/src/main/java/org/candlepin/spec/bootstrap/data/builder/Products.java

```diff
@@ -33,7 +35,7 @@ private Products() {
 
     public static ProductDTO random() {
         return new ProductDTO()
-            .id(StringUtil.random("test_id"))
+            .id(String.valueOf(new Random().nextInt(1000)))
```

Comment:
The product id has to be an number or it causes issues with the certificate encoding.

https://github.com/candlepin/candlepin/blob/master/spec/candlepin_scenarios.rb#L66

Reply:
If my PR happens to get merged first, there's a new utility method for this in `Products.randomSKU()` that may be of value here.

Reply:
Yeah, I think using your new utility method is a better solution!